Client/Frontend/Browser/BackForwardListViewController.swift

```diff
@@ -114,8 +114,9 @@ class BackForwardListViewController: UIViewController, UITableViewDataSource, UI
         self.currentItem = bfList.currentItem
 
         //error url's are OK as they are used to populate history on session restore.
-        listData = items.filter({return !($0.url.isLocal && ($0.url.originalURLFromErrorURL?.isLocal ?? true))})
-
+        
+        // disable filter local to manage Mozilla Firefox Start Page
+        listData = items.filter({return !($0.url.isLocal && ($0.url.originalURLFromErrorURL?.isLocal ?? true)) || $0.url.isAboutHomeURL})
```

Comment:
Nit: Just a personal preference but instead of using a block here I would declare a small function just above and give it a name like `homeAndNormalPagesOnly` so when future contributors come across this it's more clear what's going on here.